### PR TITLE
feat: added toast

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created tow types of toast:
Simple Toast

index 1

Stacked Toast

index 2

### CODE REVIEW

**Review by vivekmehto — APPROVED** [review]
everything looks good to me color contrast is also nice.

**Review by Sameony — APPROVED** [review]
clean and ordered structured code, very readable. good work, you can merge it

File: documentation/toast.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+
+    <link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/stackoverflow-dark.min.css" />
+
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../../stylesheets/main.css">
+    <link rel="stylesheet" href="../../stylesheets/docs.css">
+
```
Comment (shuntzhere) [inline documentation/toast.html:20]: Check if these can be imported to your main css file

**Review by shuntzhere — APPROVED** [review]
Looks good, suggested a minor thing. Can be merged.

File: documentation/toast.html (hunk 1 above)
Comment (nakulsharma15) [reply documentation/toast.html:20]: Ok, I'll check it

**Pull request merged**